tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -666,7 +668,7 @@ class ScopedActivationDescriptor {
         mode = CUDNN_ACTIVATION_TANH;
         break;
       default:
-        LOG(ERROR) << "unrecognized activation mode: "
+        LOG(FATAL) << "unrecognized activation mode: "
```

Comment:
Good point, this class is not written very well.  One day we should probably refactor this so that we can check the status rather than just kill the program.